src/charm.py

```diff
@@ -26,6 +29,9 @@
 KUBELET_KUBECONFIG_PATH = Path("/root/cdk/kubeconfig")
 KUBEPROXY_KUBECONFIG_PATH = Path("/root/cdk/kubeproxyconfig")
 
+OBSERVABILITY_GROUP = "system:cos"
+JobConfig = namedtuple("JobConfig", ["name", "metrics_path", "scheme", "target"])
```

Comment:
Maybe this is just me -- but I prefer dataclasses to namedtuples.  I think that's because i simply love python typing

```suggestion
@dataclass
class JobConfig:
    name: str
    metrics_path: str
    scheme: str
    target: str
```